core/src/main/java/io/questdb/cairo/wal/WalWriter.java

```diff
@@ -167,7 +167,7 @@ public long apply(AlterOperation operation, boolean contextAllowsAnyStructureCha
                     }
                 }
 
-                txn = tableSequencerAPI.nextStructureTxn(tableName, getStructureVersion(), operation);
+                txn = tableSequencerAPI.nextStructureTxn(tableName, (int) metadata.getStructureVersion(), operation);
```

Comment:
why did we change structure version from `long` to `int` ?

Reply:
I reverted, it is `long` again.